### PR TITLE
Hotfix: Array forEach polyfill

### PR DESCRIPTION
**Pull request by sebworks**
Hotfix: Fix for Array forEach method used in vendor dependency. Causing errors on IE8.

## Changes
- Updated  '.eslintrc' to add Array extension to no-extend-native rule.
- Added    'src/static/js/modules/array-foreach-polyfill.js' to create Array forEach polyfill.
- Added polyfill dependency to src/static/js/modules/jquery/cf_formValidator.js

## Test
- Navigate to 'http://localhost:7000/the-bureau/ ' using IE8

## Review 
- @jimmynotjim  
- @anselmbradford  
- @KimberlyMunoz 

## Screenshots

Desktop
ie

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -212,7 +212,10 @@ rules:
   no-eval: 2
 
   # Disallows adding to native types.
-  no-extend-native: 2
+  no-extend-native:
```
Comment (anselmbradford) [inline .eslintrc]: I don't think we should be adjusting the global ESLint settings for the quirks of polyfills. 
How about creating a `/src/static/js/modules/polyfill/` path where the `array-foreach-polyfill.js` module would live and put a `.eslintrc` file in `/src/static/js/modules/polyfill/.eslintrc`with the following contents:

```
# ESLint overrides for polyfills.
no-extend-native:
  - 2
  - exceptions:
    - Array
```

Comment (sebworks) [inline .eslintrc]: sounds good @anselmbradford

**Comment by jimmynotjim** [comment]
Everything is looking good :+1:

**Comment by sebworks** [comment]
@anselmbradford please review when you get a chance.

**Comment by anselmbradford** [comment]
Haven't been able to check in IE8, but code looks good :+1: Is this worth a patch commit in Changelog?

**Comment by sebworks** [comment]
Yep, will add to the Changelog.

**Comment by jimmynotjim** [comment]
This could have just been merged and then included in the minor release in the open beta PR. Probably still worth doing that so we're not changing the release tag on @dpford while he's gone.

**Comment by sebworks** [comment]
ok, will remove the commit then merge.

**Pull request merged**